### PR TITLE
Fixes Imperial Guard armor

### PR DESCRIPTION
**Pull request by ondrej008**
## About The Pull Request

I hate the person that put the m_ prefix into the held .dmi, but not the worn .dmi

## Why It's Good For The Game

armor isn't invisible anymore

## Changelog
:cl:
fix: haha funny imperial ERT now has armor (shocking)
/:cl:

### CODE REVIEW

**Comment by Hughgent** [comment]
Yo, got a picture of this working?

**Comment by ondrej008** [comment]
> 
> 
> Yo, got a picture of this working?

obrazek

File: code/modules/clothing/suits/marine_armor.dm [hunk 1]
```diff
@@ -408,15 +408,15 @@
 	name = "\improper Imperial Guard flak armour"
 	desc = "A cheap, mass produced armour worn by the Imperial Guard, which are also cheap and mass produced. You can make out what appears to be <i>Cadia stands</i> carved into the armour."
 	icon_state = "m_guardarmor"
-	item_state = "m_guardarmor"
+	item_state = "fuck_whoever_broke_my_fucking_armor"
```
Comment (TiviPlus) [inline code/modules/clothing/suits/marine_armor.dm:411]: hmmmmmmm

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/modules/clothing/suits/marine_armor.dm (hunk 1 above)
Comment (ondrej008) [reply code/modules/clothing/suits/marine_armor.dm:411]: do I have to change this

Comment (TiviPlus) [reply code/modules/clothing/suits/marine_armor.dm:411]: yes

**Pull request closed without merging**